### PR TITLE
Update msfvenom

### PR DESCRIPTION
**Pull request by DrDinosaur**
Grammar and style.

### CODE REVIEW

**Comment by hmoore-r7** [comment]
Hi @DrDinosaur. As a rule we don't enforce punctuation on comments and we try to keep the line length of printed options smaller even at the cost of grammar. Some of these cosmetic updates look useful (comment sentence capitalization is nice), but others go against the intent (smaller line length in help output).  Related are #5028 & #5026

File: msfvenom [hunk 1]
```diff
@@ -117,7 +117,7 @@ require 'msf/core/payload_generator'
       opts[:keep] = true
     end
 
-    opt.on('--payload-options', "List the payload's standard options") do
+    opt.on('--payload-options', 'List the payload's standard options') do
```
Comment (todb-r7) [inline msfvenom]: Given the interior `'` this will break formatting.

File: msfvenom [hunk 2]
```diff
@@ -66,7 +66,7 @@ require 'msf/core/payload_generator'
       end
     end
 
-    opt.on('-l', '--list       [module_type]', Array, 'List a module type example: payloads, encoders, nops, all') do |l|
```
Comment (todb-r7) [inline msfvenom]: This change looks like a keeper, since the help is worded weirdly.

**Comment by todb-r7** [comment]
+1 to HD's comments -- I don't expect to spend a lot of effort fixing punctuation and caps in inline comments, especially non-YARD formatted comments, since it'll never really be consumed by an API reference doc or anything.

I expect to pull in a couple of these changes, just trying to figure out the best/easiest way to land only a little of this PR with some creative merging, reverting, and `--author` rewriting. :)

I like #5026 and #5028 though. Will review those as well.

**Comment by todb-r7** [comment]
Note that the merged version now conflicts with this branch, @DrDinosaur you're free to delete the branch on your side.

**Pull request closed without merging**